easybuild/easyblocks/generic/makecp.py

```diff
@@ -74,9 +74,17 @@ def install_step(self):
                     # ([src1, src2], targetdir)
                     if len(fil) == 2 and isinstance(fil[0], list) and isinstance(fil[1], basestring):
                         files_specs = fil[0]
+                        new_names = None
                         target = os.path.join(self.installdir, fil[1])
+                    # ([src1, src2], [src1_new, src2_new], targetdir)
+                    elif len(fil) == 3 and isinstance(fil[0], list) and isinstance(fil[0], list) and \
```

Comment:
you're checking `fil[0]` twice

also, both `list` and `tuple` are valid as types?